waspc/data/Cli/templates/skeleton/tsconfig.json

```diff
@@ -5,6 +5,7 @@
 // compiler. Proper TS compiler configuration in Wasp is coming soon :)
 {
   "compilerOptions": {
+    "module": "esnext",
```

Comment:
I believe the latest version of Vite uses `ES2020`. If we're using Vite, should we use that one too?

The same question applies for the `target` field. I set that one to `esnext`, my bad.

Btw, nice work remembering to update OpenSaas and starters. Just link those commits/PRs in the description please :)

Reply:
Our web app uses `esnext` and IMHO it makes sense to keep them in sync: https://github.com/wasp-lang/wasp/blob/main/waspc/data/Generator/templates/react-app/tsconfig.node.json#L5

Vite uses `esnext` as well 🤷 

It makes sense TBH since users won't get IDE errors for new things and we can update our runtime over time without asking them to update their `tsconfig.json`.

I'll link to commits 👍